adapters/oidc/spring-boot-adapter-core/src/main/java/org/keycloak/adapters/springboot/KeycloakBaseSpringBootConfiguration.java

```diff
@@ -105,7 +134,7 @@ public void customize(DeploymentInfo deploymentInfo) {
             return undertowSecurityConstraints;
         }
     }
-
+    
```

Comment:
Could you keep avoid unnecessary changes to the file?

Reply:
Done. Should be better now.